docs/x509.rst

```diff
@@ -181,6 +181,76 @@ X.509 Certificate Object
 
         For version 3 X.509 certificates.
 
+.. class:: NameAttribute
+
+    .. versionadded:: 0.8
+
+    An X.509 name consists of a list of NameAttribute objects.
+
+    .. attribute:: oid
+
+        :type: :class:`ObjectIdentifier`
+
+        The attribute OID.
+
+    .. attribute:: value
+
+        :type: :term:`text`
+
+        The value of the attribute.
+
+.. class:: ObjectIdentifier
+
+    .. versionadded:: 0.8
+
+    Object identifiers (frequently seen abbreviated as OID) identify the type
+    of a value (see: :class:`NameAttribute`).
+
+    .. attribute:: value
+
+        :type: :class:`str`
```

Comment:
This will never have unicode (these are just string values of the dotted representation. The `text` discussion is for the values of Names since they can contain unicode). Everywhere else in our docs (e.g. `name` on cipher algorithms) we call this type `str`.